homework/homework.go

```diff
@@ -0,0 +1,89 @@
+package main
+
+import (
+	"fmt"
+	"gocourse05/core/camera"
+	"math/rand/v2"
+)
+
+type IDer interface {
+	ID() int
+}
+
+type Metadater interface {
+	MetaData() string
+}
+
+func main() {
+	cameras := buildCameras()
+
+	data := collectData(&cameras)
+
+	dataToServer := sendDataToServer(data)
+
+	if dataToServer == "" {
+		fmt.Println("Nothin to send, empty request")
+	}
+}
+
+func collectData(cameras *[]camera.Camera) map[int]string {
```

Comment:
Пропоную зробити так, щоб працювало наступне:
```suggestion
func collectData(cameras []camera.Camera) map[int]string {
```

Reply:
ок, зробив